src/cockpit/packages.py

```diff
@@ -298,3 +307,19 @@ def get_bridges(self):
         for package in sorted(self.packages.values(), key=lambda package: -package.priority):
             bridges.extend(package.bridges)
         return bridges
+
+    @bus.Interface.Method()
+    def reload(self):
+        self.packages = {}
+        self.load_packages()
+
+    @bus.Interface.Method()
+    def reload_hint(self):
+        # Reloading the Shell in the browser should reload the
+        # packages.  This is implemented by having the Shell call this
+        # method whenever it starts.  The first call of this method in
+        # each session is ignored so that packages are not loaded
+        # twice right after logging in.
+        if not self.ignore_next_reload_hint:
```

Comment:
I think the purpose of this variable would be a bit more clear if it was called `saw_first_reload` or so.  Something that clearly indicates that this is used once at the start and never again.  I'd also move the comment up to the top of the class — throw in a type annotation and comment that.  Or use a docstring?

Reply:
I have renamed the flag and moved the comment to the top.